ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/translator/ASTBuilder.java

```diff
@@ -243,6 +243,7 @@ public static ASTNode literal(RexLiteral literal) {
     case INTERVAL_SECOND:
     case INTERVAL_YEAR:
     case INTERVAL_YEAR_MONTH:
+    case ROW:
```

Comment:
`ROW` is missing in the type switch in L267. AFAIK it makes sense because ROW type could only be a literal if it is NULL. However, can we add it at the end of the switch with a short comment so we recall this (e.g., as it was done with `BINARY`)?